modules/distribution/src/repository/resources/dbscripts/apimgt/oracle.sql

```diff
@@ -393,10 +395,10 @@ CREATE OR REPLACE TRIGGER SP_ROLE_MAPPING_TRIG
 ALTER TABLE SP_ROLE_MAPPING ADD CONSTRAINT ROLEID_APPID_CONSTRAINT FOREIGN KEY (APP_ID) REFERENCES SP_APP (ID) ON DELETE CASCADE
 /
 CREATE TABLE SP_REQ_PATH_AUTHENTICATOR (
-            ID INTEGER,
-            TENANT_ID INTEGER NOT NULL,
-            AUTHENTICATOR_NAME VARCHAR (255) NOT NULL ,
-            APP_ID INTEGER NOT NULL,
+	    	ID INTEGER,
+	    	TENANT_ID INTEGER NOT NULL,
+	    	AUTHENTICATOR_NAME VARCHAR (255) NOT NULL ,
+	    	APP_ID INTEGER NOT NULL,
```

Comment:
Directly copied content from https://github.com/wso2/carbon-apimgt/tree/v5.0.4/features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql as apim version upgraded to v5.04